goodkey/good_key.go

```diff
@@ -44,6 +44,16 @@ type KeyPolicy struct {
 	AllowECDSANISTP521 bool // Whether ECDSA NISTP521 keys should be allowed.
 }
 
+// NewKeyPolicy returns a KeyPolicy that allows RSA, ECDSA256 and ECDSA384.
+func NewKeyPolicy() KeyPolicy {
+	return KeyPolicy{
+		AllowRSA:           true,
+		AllowECDSANISTP256: true,
+		AllowECDSANISTP384: true,
+		AllowECDSANISTP521: false,
```

Comment:
Since this is hard coded the code path at L182 will never be executed, it and the field on `KeyPolicy` should just be deleted.